submissions/UA-girl/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+        'or the cat may be completely hairless. Whiskers may be present, either whole or broken, or may be totally absent. ' +
+        'The cats have a narrow, long head and webbed feet. Their skin is the color that their fur would be, and all ' +
+        "the usual cat markings (solid, point, van, tabby, tortie, etc.) may be found on the Sphynx cat's skin. " +
+        "Because they have no fur they lose more body heat than coated cats which makes them warm to the touch and prone to finding warm places...",
+        link: 'https://en.wikipedia.org/wiki/Sphynx_cat',
+        image: {
+            url: './img/canadian_sphinx.jpg',
+            alt: 'Canadian Sphynx cat image'
+        }
+    },
+
+    {
+        id: 4,
+        title: 'Turkish Angora',
+        text: "The Turkish Angora (Turkish: Ankara kedisi, 'Ankara cat') is a breed of domestic cat. Turkish Angoras " +
+        "are one of the ancient, natural breeds of cat, having originated in central Turkey, in the Ankara region. " +
+        "The breed has been documented as early as the 17th century and is believed to be the origin of the mutations " +
+        "for both the color white[citation needed] and long hair[citation needed]. " +
+        "The breed is also sometimes referred to as simply the Angora or Ankara cat...",
+        link: 'https://en.wikipedia.org/wiki/Turkish_Angora',
+        image: {
+            url: './img/angorra.jpg',
+            alt: 'Angorra cat image'
+        }
+    },
+];
+
+const menu = document.querySelector('.catalog-list');
+const menuItems = document.querySelectorAll('.catalog-link');
+const card = document.querySelector('.card');
+
+// menuItems.forEach(item => item.addEventListener('click', toggleCatalog));
+menu.addEventListener('click', toggleCatalog);
+menu.addEventListener('keyup', function (event) {
+    if (event.keyCode === 32) {
+        toggleCatalog(event);
+    }
+});
+
+// Animate appearance of the card
+function animate(item) {
+    let opacity = +item.style.opacity;
+    if (opacity < 1) {
+        opacity += 0.05;
+        item.style.opacity = opacity.toString();
+        setTimeout(animate, 50, item);
+    }
+}
+
+
+// Fill card
+function createPattern(item, node) {
+    // Update image
+    const img = node.querySelector(`.${IMG_CLASS_NAME}`);
+    img.setAttribute('alt', item.image.alt);
+    img.setAttribute('src', item.image.url);
+    // Update data
+    const caption = node.querySelector(`.${CARD_DESCRIPTION_CLASS_NAME}`);
+    caption.querySelector(`.${CARD_TITLE_CLASS_NAME}`).textContent = item.title;
+    caption.querySelector(`.${CARD_TEXT_CLASS_NAME}`).textContent = item.text;
+    caption.querySelector(`.${CARD_LINK_CLASS_NAME}`).setAttribute('href', item.link);
+    const cardItem = card.querySelector('.card-item');
+    cardItem.style.opacity = 0;
+    animate(cardItem);
+}
+
+
+// Execute on click of menu item
+function toggleCatalog(event) {
```

Comment:
It seems you can change this variable name to the some more informative name. Maybe `toggleCatDescription` would be better. Think about this in future, please